app/assets/javascripts/components/miq-button.js

```diff
@@ -0,0 +1,40 @@
+ManageIQ.angular.app.component('miqButton', {
+  bindings: {
+    name: '@',
+    enabled: '<',
+    enabledTitle: '@?',
+    disabledTitle: '@?',
+    primary: '<?',
+    onClick: '&',
+  },
+  controller: function() {
+    this.buttonClicked = function($event) {
```

Comment:
`$event` does not seem to be used at all.
If you need it around anyway, change it to `_$event`, and if not, remove it.